spec/access-identify.md

```diff
@@ -0,0 +1,303 @@
+## `access/identify`
+
+Upload protocol defines [access/identify][] capability which is very flexible yet powerful so it can be very confusing. This document attempts to wolk through some user stories to try and clarify.
+
+#### identify with email
+
+"Accounting" service MAY delegate other "verifier" service(s) to do out of bound user verification e.g. sign-in via email link style flow.
+
+To do so _accounting_ must delegate a capability to _verifier_ service which will allows it:
+
+1. To create an accounts under `did:email:` namespace.
+2. To associate other DIDs with accounts under `did:email:` namespace.
+
+:::danger
+It is important to call out that accounting needs to put a lot of trust into such service, as malicious verifier could get control of other accounts
+:::
+
+```js
+{
```

Comment:
```suggestion
```js
// The account service signs this with it's key, and sends it to the verifier service
{
```